src/main/scala/geotrellis/sdg/PopulationNearRoadsJob.scala

```diff
@@ -123,6 +116,57 @@ class PopulationNearRoadsJob(
     ContextRDD(rdd, md)
   }
 
+  def forgottenLayerTiles(outputUri: URI, pixelScale: Int): Unit = {
+    import spark.implicits._
+    val maxZoom = 12
+    val minZoom = 6
+    // Tweak this value by powers of 2 to increase or reduce the pixel size in the output
+    // Higher number == smaller pixels
+    val pixelsPerTile = math.pow(2, pixelScale).toInt
+    val wmLayoutScheme = ZoomedLayoutScheme(WebMercator, pixelsPerTile)
+    val wmLayout: LayoutDefinition = wmLayoutScheme.levelForZoom(maxZoom).layout
+    val latLngToWebMercator = Transform(LatLng, WebMercator)
+
+    val gridPointsRdd: RDD[(Long, Long, Double)] = forgottenLayer.flatMap {
+      case (key: SpatialKey, tile: Tile) => {
+        val tileExtent = key.extent(layout)
+        val re = RasterExtent(tileExtent, tile)
+        for {
+          col <- Iterator.range(0, tile.cols)
+          row <- Iterator.range(0, tile.rows)
+          v = tile.getDouble(col, row)
+          if isData(v)
+        } yield {
+          val (lon, lat) = re.gridToMap(col, row)
+          val point = Point(lon, lat)
+          val wmPoint = point.reproject(latLngToWebMercator)
+          val (x, y) = wmLayout.mapToGrid(wmPoint)
+          (x, y, v)
+        }
+      }
+    }
+
+    val pipeline = ForgottenPopPipeline(
+      "geom",
+      URI.create(s"${outputUri.toString}/x$pixelScale"),
+      wmLayoutScheme,
+      maxZoom
+    )
+    val vpOptions = VectorPipe.Options(
```

Comment:
There's an issue where at high pixelScale (>= 8) the grid at zooms >= 11 starts to have gaps in it.

I figured this might be caused by our pixel size approaching the size of the subpixel grid in the vector tile, so I increased the VectorPipe.Options.tileResolution to 8192 from 4096. This had no effect on the output. If design wants really small pixels this is something we'll have to investigate further.

![Screen Shot 2019-10-21 at 4 41 14 PM](https://user-images.githubusercontent.com/1818302/67241708-eb486880-f421-11e9-92bc-a9283818a035.png)